src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.Calculator/CalculateHelper.cs

```diff
@@ -29,7 +29,7 @@ public static bool InputValid(string input)
             }
 
             bool singleDigitFactorial = input.EndsWith("!", StringComparison.InvariantCulture);
-            if (input.Length <= 2 && !singleDigitFactorial)
+            if (input.Length <= 0 && !singleDigitFactorial)
```

Comment:
@jaimecbernardo 
Can we remove this as ut only makes problems or was there a special reason to have this?

Unfortunately there is no explaining comment.

Reply:
I've looked and it seems to come from https://github.com/microsoft/PowerToys/pull/14736
@stefansjfw , can we get your input here, please?

Reply:
@htcfreek To remove what? singleDigitFactorial part or input.Length part?